packages/backend/src/util/user.decorator.ts

```diff
@@ -0,0 +1,9 @@
+import { createParamDecorator } from '@nestjs/common';
+
+export const User = createParamDecorator((data, req) => {
+  const user = req.getArgs()[0].user;
```

Comment:
If anyone wants to investigate if there's a better way to do this then go for it :P